lib/dictionary.js

```diff
@@ -0,0 +1,23 @@
+const binding = require('../binding')
+const ReferenceCounted = require('./reference-counted')
+
+module.exports = class Dictionary extends ReferenceCounted {
```

Comment:
```suggestion
module.exports = class FFmpegDictionary extends ReferenceCounted {
```

Qualified names makes it much easier to spot where the class is coming from when looking at debug output and heap traces.